extensions/wikia/ARecoveryEngine/tests/ARecoveryModuleTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+class ARecoveryModuleTest extends WikiaBaseTest {
+	const bootstrapCode = 'BOOTSTRAP_CODE';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this constant "bootstrapCode" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS115)